gptme/message.py

```diff
@@ -213,8 +213,8 @@ def format_msgs(
             userprefix = f"[bold {color}]{userprefix}[/bold {color}]"
         if msg.role == "system":
             first_line = msg.content.split("\n", 1)[0].lower()
-            isSuccess = first_line.startswith(("saved", "executed", "ran", "appended", "patch successfully"))
-            isError = first_line.startswith(("error", "failed","missing arguments", "unknown command", "no patch"))
+            isSuccess = first_line.startswith(("saved", "appended", "patch successfully"))
+            isError = first_line.startswith(("error", "failed")) or "Return code" in msg.content[-16:]
```

Comment:
When a shell command exits with a non-zero code the code is part of the message. Only captures unsuccessful commands. Could change the code to append the return code for each command and strip the zero return codes before displaying the message.